docs/Releasing.md

```diff
@@ -27,4 +27,167 @@ After checking that everything looks good, you just need to push the changes:
 
 ```
 git push --tags
-```
\ No newline at end of file
+```
+
+# Release Checklist
+
+Just copy this checklist and replace all occurrences of `X.XX.X` with the applicable release number, when we are ready to
+cut a new release.
+
+```
+<!-- wp:heading {"level":1} -->
```

Comment:
Can we add a message here and a link to this doc, so no one is tempted to copy/paste the p2 post directly. Something like: "If you need to release gutenberg-mobile, please use [the checklist from our GitHub repo](https://github.com/wordpress-mobile/gutenberg-mobile/blob/master/docs/Releasing.md)"

@SergioEstevao, can we also add "deprecated document, please visit https://github.com/wordpress-mobile/gutenberg-mobile/blob/master/docs/Releasing.md" in the release gdoc?

Reply:
> @SergioEstevao, can we also add "deprecated document, please visit https://github.com/wordpress-mobile/gutenberg-mobile/blob/master/docs/Releasing.md" in the release gdoc?

@maxme My understanding is that this document will replace the one that you link, so were do we want to mark deprecate?

Reply:
We can delete the gdoc, I want to make sure no one update it or use it.

Reply:
> Can we add a message here and a link to this doc, so no one is tempted to copy/paste the p2 post directly.

Good idea @maxme. Added in ee027c4.